docs/deployment.rst

```diff
@@ -10,7 +10,7 @@ This chapter details deployment strategies for the following scenarios.
    :titlesonly:
 
    deployment/central-backup-server
-   deployment/hosting-repositories
+   deployment/hosting-secure-repositories
```

Comment:
i'ld try to avoid renaming stuff.

the last time, when usage_general was renamed, I had to manually deal with a lot of merge conflicts.